modules/slacker-plugin-extra/src/main/java/com/labs2160/slacker/plugin/extra/ScheduleTrigger.java

```diff
@@ -1,72 +1,46 @@
 package com.labs2160.slacker.plugin.extra;
 
-import com.labs2160.slacker.api.Trigger;
 import com.labs2160.slacker.api.RequestHandler;
 import com.labs2160.slacker.api.SlackerRequest;
 import com.labs2160.slacker.api.SlackerResponse;
-
-import java.util.concurrent.Future;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.Properties;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.Executors;
-import java.util.TimerTask;
+import com.labs2160.slacker.api.Trigger;
+import it.sauronsoftware.cron4j.Scheduler;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Properties;
+import java.util.concurrent.Future;
+
 public class ScheduleTrigger implements Trigger {
 
     private static final Logger logger = LoggerFactory.getLogger(Trigger.class);
-    private final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
-
-    private String interval;
-    private String[] requests;
+    private static final Scheduler scheduler = new Scheduler();
```

Comment:
The cron4j Scheduler should be in the engine.   There should only be 1 scheduler in the entire application.
You submit multiple jobs to the scheduler.
You can pause, shutdown, and restart the scheduler and all jobs that it manages.

We shouldn't be creating a new Scheduler for each Trigger.